lighthouse-core/gather/driver.js

```diff
@@ -25,13 +25,15 @@ const URL = require('../lib/url-shim');
 const log = require('../lib/log.js');
 const DevtoolsLog = require('./devtools-log');
 
-const PAUSE_AFTER_LOAD = 5000;
+const DEFAULT_PAUSE_AFTER_LOAD = 0;
```

Comment:
can you add comments above these to explain what they are for? otherwise folks will have to backtrack to this discussion to totally understand. :)